src/jvmMain/kotlin/io/beatmaps/api/review.kt

```diff
@@ -531,6 +551,37 @@ fun Route.reviewRoute() {
                             it[updatedAt] = NowExpression(updatedAt)
                         }.value
 
+                        if (insertedId != null) {
+                            val alertHeader = "New Review Reply"
+
+                            if (user.userId != reviewUserId) {
+                                Alert.insert(
+                                    alertHeader,
+                                    "@${user.uniqueName} just replied to your review on #${toHexString(mapId)}: **$mapName**.\n" +
+                                        "*\"${reply.text.replace(Regex("\n+"), " ").take(100)}...\"*",
+                                    EAlertType.ReviewReply,
+                                    reviewUserId
+                                )
+
+                                updateAlertCount(reviewUserId)
+                            }
+
+                            for (singleCollaborator in collaborators) {
+                                if (singleCollaborator[collaboratorAlias[User.id]].value == user.userId) continue
```

Comment:
Does this work? User.id isn't in the select (although Collaboration.collaboratorId is)
I would have just kept the map above and made it
```kt
.map { it[Collaboration.collaboratorId].value to it[collaboratorAlias[User.reviewAlerts]] }
```
then here
```kt
filter { it.first != user.Id && it.second }
```

Reply:
In fact, it doesn't as I had come to notice too :man_facepalming: 
I change the part to a filter -> forEach too. I did not map it though.

Using `first` and `second` seemed a bit too unclear to me. I hope this works too, though.